src/app/app.context.tsx

```diff
@@ -1,27 +1,28 @@
 import React, { createContext } from "react";
 import { appTypes } from "./app.types";
+import { Theme } from "../enum/theme";
 
 interface AppProviderProps {
   children: React.ReactNode;
 }
 type Dispatch = (action: any) => void;
 
 interface AppProps {
-  theme: string;
+  theme: Theme;
 }
```

Comment:
lets move interface to type file and action type to constant file of respective module

Reply:
👍